### DISCUSSION TITLE
Support for Yoast Focus Keyword, meta description and seo title

### DISCUSSION THREAD

**Discussion by jphorn** [publishpress/publishpress-checklists#530] [Feature Requests]
This is a follow-up request from an older Github issue which has since been closed: #86 

The issue was closed because you (then) chose to piggyback onto Yoast's own checks for Readability Analysis and SEO Analysis score. However, I don't think that's the correct approach as the green (or orange) bullets are a) easy to game and b) not a very good indicator and c) it's even [discouraged by team Yoast](https://yoast.com/want-green-bullet-wp-seo/).

Especially their number 1 item rings true: "1. Not every post or page has to be optimized". We have many different custom post types, and they don't all need optimisation. BUT they *might* sometimes need a focus keyword or a filled-in SEO meta description. Right now I can't make this a hard requirement in Checklists Pro, and that prevents me from a global rollout on our site. 

We always used Requirement Checklists (Classic Editor only, and now seemingly abandoned) and they had these checks. Also I noticed HumanMade/Altis Publication Checklist offer a similar check for SEO Title ([ref](https://github.com/humanmade/demo-publication-checklist/blob/02a61e26cc907c98995f4e8abff0a05d1a5954bf/inc/namespace.php#L111-L124)).

I would really love to see these separate Yoast SEO checks in Checklists (Pro) instead of (imho incorrectly) relying on the SEO assessment analysis.

Thank you for considering.

**Comment by rizaardiyanto1412** [comment]
This is already implemented https://github.com/publishpress/PublishPress-Checklists/issues/555
